DTS-STN/next-template
# Update Tailwind to v3

## [DTSCI-94 - Next Template - Tailwind Updates](https://jira-dev.bdm-dev.dts-stn.com/browse/DTSCI-94) (Jira Issue)

### Description

The purpose of this PR is to upgrade Tailwind to v3 following [the official upgrade guide](https://tailwindcss.com/docs/upgrade-guide). I've also added the tailwind typography plugin.

List of proposed changes:

- We could explore adding some default custom colors that align with the Canada.ca theme 

### What to test for/How to test

1. Pull in branch
2. Try adding tailwind classes to an element on `index.js`
3. Ensure styling still works
